Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -2346,4 +2346,17 @@ void WebsiteDataStore::resumeDownload(const DownloadProxy& downloadProxy, const
     networkProcess().send(Messages::NetworkProcess::ResumeDownload(m_sessionID, downloadProxy.downloadID(), resumeData.dataReference(), path, sandboxExtensionHandle, callDownloadDidStart), 0);
 }
 
+#if HAVE(NW_PROXY_CONFIG)
+void WebsiteDataStore::clearProxyConfigData()
+{
+    networkProcess().send(Messages::NetworkProcess::ClearProxyConfigData(m_sessionID), 0);
+}
+
+void WebsiteDataStore::setProxyConfigData(const API::Data& data, uuid_t proxyIdentifier)
+{
+    auto proxyIdentifierData = IPC::DataReference((uint8_t *)proxyIdentifier, sizeof(uuid_t));
```

Comment:
I think it's be nicer to use a C++ cast (reinterpret_cast here) rather than a C cast.